From 1c02240a3bd4f20578255ff45731905a521ae2f3 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 21 May 2025 08:51:45 +0200
Subject: [PATCH] arm64: dts: st: fix lvds support on stm32mp255

Keep same clock order than stm32mp251 to get always the lcd clock link
to the rcc clock CK_KER_LTDC

Change-Id: Iad5307d087ec580fe0579068f7476f682fbc7e39
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/460518
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp255.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp255.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp255.dtsi
@@ -29,7 +29,7 @@
 
 &ltdc {
 	clocks = <&rcc CK_BUS_LTDC>, <&rcc CK_KER_LTDC>, <&syscfg 0>, <&lvds 0>;
-	clock-names = "bus", "ref", "lcd", "lvds";
+	clock-names = "bus", "lcd", "ref", "lvds";
 };
 
 &rcc {
